upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -98,7 +97,7 @@ func FindCloudInstanceGroups(cloud fi.Cloud, cluster *api.Cluster, instancegroup
 	return groups, nil
 }
 
-func (c *RollingUpdateCluster) RollingUpdate(groups map[string]*CloudInstanceGroup, k8sClient *k8s_clientset.Clientset) error {
+func (c *RollingUpdateCluster) RollingUpdate(groups map[string]*CloudInstanceGroup, k8sClient k8s_clientset.Interface) error {
```

Comment:
So that's what we need to do ... I think we have a bunch of places this needs to be fixed.

Learned something today :)